index.html

```diff
@@ -18,52 +18,259 @@
+          <li>
+            <a href="#">
+              <img src="images/icons/twitter.svg" alt="Twitter Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/linkedin.png" alt="LinkedIn Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/m.svg" alt="Media Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/cat.svg" alt="Github Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/fingers.svg" alt="Victory Icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!--Works Section-->
+    <section class="works">
+      <div class="grid-container">
+        <div class="card-layout">
+          <a href="#">
+            <img
+              class="image-fluid"
+              src="images/works/image_1.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h2 class="title">Tonic</h2>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card-layout">
+          <a href="#">
+            <img
+              class="image-fluid"
+              src="images/works/image_2.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h2 class="title">Tonic</h2>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
```

Comment:
- [ ] Same as above and as requested by the previous code reviewer and project requirements, Kindly wrapp the languages into [```ul li```](https://github.com/microverseinc/curriculum-html-css/blob/3e39af5af8075c97c82e7fec70edd3ff40345234/portfolio/2_mobile.md?plain=1#L51-L52) tags. e.g 
```html 
<ul class="btn-container-flex">
  <li type="button" class="btn-sm">html</li>
  <li type="button" class="btn-md">css</li>
  <li type="button" class="btn-lg">javaScript</li>
</ul>
```
then polish up the UI to match the design, should this change affect how awesome your buttons currently look.

Reply:
As mentioned above, Kindly note that The code snippet was just an example, for your reference. However, for list items - the `type` attribute is not needed.

Reply:
> * [ ]  Same as above and as requested by the previous code reviewer and project requirements, Kindly wrapp the languages into [`ul li`](https://github.com/microverseinc/curriculum-html-css/blob/3e39af5af8075c97c82e7fec70edd3ff40345234/portfolio/2_mobile.md?plain=1#L51-L52) tags. e.g
> 
> ```
> <ul class="btn-container-flex">
>   <li type="button" class="btn-sm">html</li>
>   <li type="button" class="btn-md">css</li>
>   <li type="button" class="btn-lg">javaScript</li>
> </ul>
> ```
> 
> then polish up the UI to match the design, should this change affect how awesome your buttons currently look.

@blessedjasonmwanza from my observation the colour I left out in the languages buttons is the active state of Enabled which is  #fff which I have changed. I seem to have assigned similar colours to all the other buttons. kindly confirm if this is the colour you were referring to.